tools/crm_resource_runtime.c

```diff
@@ -221,7 +221,8 @@ find_matching_attr_resources(pcmk__output_t *out, pe_resource_t * rsc,
         if(rc != pcmk_rc_ok) {
             rsc = rsc->parent;
             if (!out->is_quiet(out)) {
-                printf("Performing %s of '%s' on '%s', the parent of '%s'\n", cmd, attr_name, rsc->id, rsc_id);
+                out->info(out, "Performing %s of '%s' on '%s', the parent of '%s'",
+                          cmd, attr_name, rsc->id, rsc_id);
```

Comment:
Knowing that a different resource was used (rsc->id) is potentially useful to have in XML. It looks like extracting that would be a pain though, so I wouldn't worry about it unless demand arises in the future